src/main/java/com/jvm_bloggers/core/data_fetching/blogs/domain/Blog.java

```diff
@@ -25,7 +26,7 @@
 @Builder
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-public class Blog {
+public class Blog implements Serializable {
```

Comment:
Because I used `Model.of(Serializable)` in tests, I will try to use some other approach